extensions/wikia/ArticleComments/classes/ArticleCommentInit.class.php

```diff
@@ -41,6 +41,8 @@ static public function ArticleCommentCheck( $title = null ) {
 
 	/**
 	 * Check whether comments should be enabled for given title
+	 * @param Title $title
+	 * @return bool
 	 */
 	static public function ArticleCommentCheckTitle( $title ) {
```

Comment:
Are you planning to add strong typing to parameters as you're adding them to docs?